wagtail/admin/templates/wagtailadmin/reports/listing/_list_publish.html

```diff
@@ -0,0 +1,32 @@
+{% extends "wagtailadmin/reports/listing/_list_page_report.html" %}
+
+{% load i18n wagtailadmin_tags %}
+
+{% block extra_columns %}
+    <th>{% trans 'Scheduled by' %}</th>
+    <th>{% trans 'Scheduled to be published' %}</th>
+{% endblock %}
+
+
+{% block extra_page_data%}
+    <td valign="top">
+        {{ page.get_latest_revision.user }}
+    </td>
+    <td valign="top">
+        {% page_permissions page as perms %}
+        <p>
+            {% with page.get_latest_revision.approved_go_live_at|date:"DATETIME_FORMAT" as locking_date %}
+                {% blocktrans %}
```

Comment:
Nit: There's no need to translate a date.

Reply:
This is a format used in the template for locked pages. We simply copied it at the time.

Reply:
OK, this was copied from similar templates in the reports section of the templates. Don't really care either way.

Reply:
OK I see your point now, the original template probably had more than the date in the translation block :)